include/roboteam_ai/treeinterp/BTFactory.h

```diff
@@ -33,6 +33,8 @@ class BTFactory {
 
         static std::map<std::string, bt::BehaviorTree::Ptr> keeperRepo;
 
+        static std::map<std::string, bt::BehaviorTree::Ptr> codeTrees;
```

Comment:
How many fields will this codeTrees / keeperRepo have? If it's more han like 50 i'd use unordered p sure, would have to benchmark